routes/api/v1/meals.js

```diff
@@ -31,7 +30,26 @@ var getAllFoods = (food) => {
      calories: food.calories}
 }
 
+const existingMealById = (mealId) => {
+  return Meal.findOne({ where: {id: mealId}})
+  .then(meal => {
+    if (meal != null) { return meal } else { return false }
```

Comment:
I've experimented with this a little bit, and we could get rid of the !=null, unless we want to leave it for clarity.